plugins/main/public/controllers/agent/agents.js

```diff
@@ -63,6 +63,9 @@ export class AgentsController {
     this.csvReq = csvReq;
     this.wazuhConfig = new WazuhConfig();
     this.genericReq = GenericRequest;
+    this.$scope.$watch('tab', function (newValue, oldValue) {
+      console.log('tab changed', { newValue, oldValue });
+    });
```

Comment:
**nitpick**: remove this code.

Reply:
Fixed on https://github.com/wazuh/wazuh-dashboard-plugins/pull/6005/commits/cd628a40b7cc496ecc9ecd56e76313d5a9b2ce3f.